distribution/docs/src/main/jdocs/content/_security-framework/security-rest-intro-contents.adoc

```diff
@@ -0,0 +1,80 @@
+|       |                   |SAML Assert|  |                      |  |  AuthZ  |   |       |       |cDEF    |   /=--------\                                  |
+|       |                   |  Handler  |  |                      \->| Filter  |-->|----=->|------>|        |-->|         |                                  |
+|       |   /--------\      \-----------/  |      /-----------\      |/-------\|   |       |       |  PDP   |   |Expansion|                                  |
+|       |   | cF55   |            |        |      | cF55      |  No  || Has   ||   |       |       |        |   | Service |                                  |
+|       |   | Not    |            v        |      | Not       |<-----|| Perm? ||   |       |       |        |   |         |                                  |
+|       |   |Handled |      /-----------\  |      | Authorized|      |\-------/|<--|<----=-|<------|        |<--|         |                                  |
+|       |   \--------/      |   Other   |  |      \-----------/      |    |Yes |   |       |       |        |   \---------/                                  |
+|       |       ^           | Handlers  |  |                         |    v    |   |       |       \--------/                                                |
+|       |       |           \-----------/  |                         \---------/   \-------/                                                                 |
+|       |       |               |   |      |                              |                                                                                  |
+|       |       \---------------/   \------/                              v                                                                                  |
+|       |                                       Response        /------------------\                                                                         |
+|       \-------------------------------------------------------|c5F5 Endpoint     |                                                                         |
+|                                                               \------------------/                                                                         |
++------------------------------------------------------------------------------------------------------------------------------------------------------------+
+....
+
+The Delegate Servlet Filter is the topmost filter for all web contexts.
+It initializes all Security Filters and runs them in order according to service ranking:
+
+. The *Web SSO Filter* reads from the web context policy manager and functions as the first decision point.
+If the request is from a whitelisted context, no further authentication is needed and the request goes directly to the desired endpoint.
+If the context is not on the whitelist, the filter will attempt to get a handler for the context.
```

Comment:
❓ What kind of "handler"? Claims handler? Or a different kind of handler?

Reply:
claims handler, I believe